CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Removed
 
 Fixed
 -----
+- Bug in handling parallelization in matrix_product_state.cpp
```

Comment:
We need to add the number of the PR at the end of this line (see the other entries) :)

Reply:
right. done.